### PR TITLE
add new button variant

### PR DESCRIPTION
**Pull request by JefMari**
For issue #162 
image

### CODE REVIEW

**Comment by JefMari** [comment]
Please Check @srph

**Comment by srph** [comment]
`-primary-link` has a border. It's the `-default` with a primary-light color.

Also, does `-primary-link` fit? cc @maebe22 @vistajess @juztinlazaro

**Comment by JefMari** [comment]
Border applied!
image

Question: How about its hover property? currently it also adopts `default` button hover property

**Comment by srph** [comment]
Ask a designer. IMO, darken the text.

**Comment by JefMari** [comment]
Update! 
image

**Comment by srph** [comment]
Was the darkened bg intentional?

**Comment by JefMari** [comment]
nah, the text color only darken, though the bg is inherited from `default`

**Comment by srph** [comment]
Can you ask Lemon?

**Comment by JefMari** [comment]
Ok! :100:

**Comment by JefMari** [comment]
I ask sir lemon about the proper design, and this is what he wants.

image

hovered

image

**Comment by srph** [comment]
What about the button in the checkut?

**Comment by JefMari** [comment]
Update!
image
hovered
image

File: styles/components/_button.scss [hunk 1]
```diff
@@ -53,7 +53,7 @@
     border: 1px solid $brand-gray;
   }
 
-  &.-primary { @include btn-variant($brand-sidebar-lt); }
+  &.-primary { @include btn-variant($brand-primary); }
```
Comment (srph) [inline styles/components/_button.scss]: I think there's a huge misunderstanding. Last time, we talked about changing the primary button to `$brand-sidebar-lt`. Why the change again? :/

Comment (JefMari) [inline styles/components/_button.scss]: woops

**Comment by JefMari** [comment]
Update!
image

Question: should it also applied in `inverted` colors?

**Comment by srph** [comment]
No. Also, we might remove the inverted soon (but it has nothing to do with this).

**Comment by JefMari** [comment]
Noted! Gonna merged this na

File: styles/components/_button.scss [hunk 2]
```diff
@@ -76,8 +76,19 @@
...
+    &:hover,
+    &:focus {
+      color: $brand-white;
+      background: $brand-sidebar-lt;
+    }
+  }
+
   &.-inverted.-default { @include btn-variant-inverted($brand-gray); }
-  &.-inverted.-primary { @include btn-variant-inverted($brand-sidebar-lt); }
+  &.-inverted.-primary { @include btn-variant-inverted($brand-primary); }
```
Comment (srph) [inline styles/components/_button.scss]: Why?

File: styles/components/_button.scss [hunk 3]
```diff
@@ -76,8 +76,19 @@
...
 
+  &.-primary-link { 
+    @include btn-variant($brand-white, $brand-sidebar-lt);
+    border: 1px solid $brand-gray;
+
+    &:hover,
+    &:focus {
+      color: $brand-white;
+      background: $brand-sidebar-lt;
+    }
```
Comment (srph) [inline styles/components/_button.scss:87]: So bordered -> bg? Hmm, OK.

Comment (JefMari) [inline styles/components/_button.scss:87]: Should i also change it?

Comment (srph) [inline styles/components/_button.scss:87]: I have no solid opinion about this. But I'm OK with it :ok_hand:. Anyway, was this instructed or did you come up with it?

**Comment by srph** [comment]
What's with the new color change?

File: styles/components/_button.scss (hunk 3 above)
Comment (JefMari) [inline styles/components/_button.scss:87]: It was instructed by Sir lemon

**Comment by JefMari** [comment]
I also did change of color in `inverted`, so i change it back :D

**Comment by JefMari** [comment]
I also did change of color in inverted, so i change it back :D

image

**Comment by srph** [comment]
OK. Like I said, I have no solid opinion about this. I'd suggest you ask sir Lemon again if you're out of ideas ahah. Thanks
